osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/SetAlarmNotificationsCommandExecutor.java

```diff
@@ -82,20 +83,25 @@ public AccessResultCode execute(
       final MessageMetadata messageMetadata)
       throws ProtocolAdapterException {
 
-    final AttributeAddress alarmFilter1AttributeAddress =
-        this.dlmsObjectConfigService.getAttributeAddress(
-            device, DlmsObjectType.ALARM_FILTER_1, null);
+    final Protocol protocol = Protocol.forDevice(device);
+    AccessResultCode resultCodeAlarmFilter1 = null;
 
-    final AccessResultCode resultCodeAlarmFilter1 =
-        this.setAlarmNotifications(
-            conn,
-            alarmNotifications,
-            alarmFilter1AttributeAddress,
-            DlmsObjectType.ALARM_REGISTER_1);
+    final Optional<AttributeAddress> alarmFilter1AttributeAddress =
+        this.objectConfigServiceHelper.findOptionalDefaultAttributeAddress(
```

Comment:
Volgens mij hoef je hier niet findOptionalDefaultAttributeAddress hier kun je ervan uit gaan dat het object bestaat, net als in de oude code

Reply:
Klopt, is nu geen Optional meer.